src/components/truncated-text/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {
+  memo,
+  FC,
+  PropsWithChildren,
+  useState,
+  useRef,
+  useEffect
+} from 'react';
+
+import useResize from './hooks';
+
+import SC from './styled';
+
+interface Props {
+  content: string;
+  visibleLines: number;
+  lineHeight: number;
+}
+
+const TruncatedText: FC<PropsWithChildren<Props>> = ({
+  content,
+  visibleLines,
+  lineHeight
+}) => {
+  const [expanded, setExpanded] = useState(false);
+  const [truncate, setTruncated] = useState(false);
+
+  const ref = useRef<HTMLDivElement>(null);
+  const { height } = useResize(ref);
+
+  useEffect(() => {
+    setTruncated(height / lineHeight > visibleLines);
+  }, [height]);
+
+  return (
+    <div>
+      <SC.TextContent
+        ref={ref}
+        lineHeight={lineHeight}
+        truncate={truncate && !expanded}
+        visibleLines={visibleLines + 1}
+      >
+        {content}
+      </SC.TextContent>
+      {truncate && (
+        <SC.ExpandButton onClick={() => setExpanded(!expanded)} open={expanded}>
+          {expanded ? 'Skjul beskrivelsen' : 'Vis hele beskrivelsen'}
```

Comment:
Legge tekstene inn i språkfilene (nb, nn og en) slik at du viser basert på valgt språk. Mulig at samme ledetekst allerede ligger der.

Reply:
Fikset det nå - men for å få endring i locale til å re-rendre teksten måtte jeg fjerne `React.memo` fra `ContentSection`. Her finnes det kanskje en annen løsning? Ellers funker det som det er nå!